pkg/controller/common/keystore/initcontainer.go

```diff
@@ -27,6 +27,8 @@ type InitContainerParameters struct {
 	KeystoreAddCommand string
 	// Keystore create command
 	KeystoreCreateCommand string
+	// ContainerCommand is the bash script to run in container
+	ContainerCommand string
```

Comment:
```suggestion
	CustomScript string
```

This may be easier to follow, if we call this `CustomScript`, and comment to state that this is a script to run that overrides the default Keystore script?